bundles/org.openhab.binding.astro/README.md

```diff
@@ -57,7 +57,7 @@ This is done by setting `useMeteorologicalSeason` to true in the advanced settin
       - `totalElevation, partialElevation, ringElevation` (Number:Angle)
   - **group** `phase`
     - **channel**
-      - `name` (String), values: `SUN_RISE, ASTRO_DAWN, NAUTIC_DAWN, CIVIL_DAWN, CIVIL_DUSK, NAUTIC_DUSK, ASTRO_DUSK, SUN_SET, DAYLIGHT, NIGHT`
+      - `name` (String), values: `SUN_RISE, ASTRO_DAWN, NAUTIC_DAWN, CIVIL_DAWN, CIVIL_DUSK, NAUTIC_DUSK, ASTRO_DUSK, SUN_SET, DAYLIGHT, NOON, NIGHT, MORNING_NIGHT, EVENING_NIGHT`
```

Comment:
I do not find MORNING_NIGHT and EVENING_NIGHT in channel type description:
https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.astro/src/main/resources/OH-INF/thing/channels.xml#L283

Reply:
They are present in the java enum :https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.astro/src/main/java/org/openhab/binding/astro/internal/model/SunPhaseName.java

Reply:
I'm not sure, if they are used actually and if yes, I've no idea when. Tried to trigger it manually (by changing locations), but I couldn't: `DAYLIGHT` is followed by `SUN_SET` and the different `*_DUSK` then before `NIGHT` but apparently no `EVENING_NIGHT` between (or the interval is so short, that my manual tries didn't match it).

So we have two options here:
1. also add it to the channel type descriptions, or
2. remove them in the README again

WDYT?

Reply:
If they are not fired, I would suggest to remove them from documentation.

Reply:
Done, even though I'm not sure if they could eventually trigger in some other configuration...